### PR TITLE
Added initial exercise drafts

### PR DESCRIPTION
**Pull request by philippjfr**
Still need to work out a lot of the instructions and the exercise for 5.

### CODE REVIEW

**Comment by jbednar** [comment]
Maybe rename all the notebooks to start like ``Ex2-``?  They are all in ``exercises``, so they shouldn't be ambiguous.

File: exercises/Exercise-2-datasets-and-collections-of-data.ipynb [hunk 1]
```diff
@@ -231,37 +259,47 @@
...
+   },
    "outputs": [],
    "source": []
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<b><a href=\"#solution6\" data-toggle=\"collapse\">Solution</a></b>\n",
+    "<b><a href=\"#solution7\" data-toggle=\"collapse\">Solution</a></b>\n",
```
Comment (philippjfr) [inline exercises/Exercise-2-datasets-and-collections-of-data.ipynb]: Something is off here, the link id here must match the id below.

Comment (jbednar) [reply exercises/Exercise-2-datasets-and-collections-of-data.ipynb]: Thanks. There were several in this file that were mixed up like that previously, so I had to renumber things overall, but it looks like I needed to renumber one more. Fixed now.

**Comment by jbednar** [comment]
Ok, I've gone through them all, and am happy for them to be merged.  Good job!  Do you have a version of exercise 5 in draft form?

**Pull request merged**